src/core/lib/support/cpu_linux.c

```diff
@@ -67,12 +67,17 @@ unsigned gpr_cpu_num_cores(void) {
 }
 
 unsigned gpr_cpu_current_cpu(void) {
+#ifdef __GLIBC__
   int cpu = sched_getcpu();
   if (cpu < 0) {
     gpr_log(GPR_ERROR, "Error determining current CPU: %s\n", strerror(errno));
     return 0;
   }
   return (unsigned)cpu;
+#else
+  // sched_getcpu() is undefined on musl
```

Comment:
I thought musl libc 1.1.15 (July 2016) added it.  Is it not functional for gRPC's use?